<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewReel</class>
 <widget class="QDialog" name="NewReel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Reel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Reel Number:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtReelNumber"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Supplier Reel Number:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtSupplierReelNumber"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Supplier:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cboSupplier">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Chandaria</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mufindi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mondi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PPM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ESS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Royal</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cboType">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>WK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>F</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TL</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cboSize">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>GSM:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cboGSM">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>112</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>125</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>127</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>135</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>140</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>150</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>160</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>175</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Opening Weight:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="txtOpeningWeight"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>As Of:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDateEdit" name="dtpAsOf">
       <property name="displayFormat">
        <string>dd MMM yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>92</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdCancel</sender>
   <signal>clicked()</signal>
   <receiver>NewReel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
